### PR TITLE
Improve version checking code

### PR DESCRIPTION
**Pull request by jla524**
1. Use the first part of sys.version to print Python version instead of converting and adding parts of sys.version_info
2. To check Python version, we can simply use a tuple comparison

### CODE REVIEW

File: sherlock/__main__.py [hunk 1]
```diff
@@ -11,18 +11,10 @@
...
 if __name__ == "__main__":
-    # Checking if the user is using the correct version of Python
-    # Reference:
-    #  If Python version is 3.6.5
-    #               major --^
-    #               minor ----^
-    #               micro ------^
-    major = sys.version_info[0]
-    minor = sys.version_info[1]
+    """Check if the user is using the correct version of Python"""
```
Comment (jcs090218) [inline sherlock/__main__.py:14]: You should probably use `#` for comment and not `"""`.

Comment (jla524) [reply sherlock/__main__.py]: Okay, I've changed it back to #.

**Comment by donno2048** [comment]
Actually, I have already opened a PR for this exact purpose #1032

**Comment by jla524** [comment]
> Actually, I have already opened a PR for this exact purpose #1032

@donno2048 Just so you know, Python has an [operator precedence](https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html) which always evaluates the `and` operator before `or`. Therefore, running the current version of the script with Python 2.7 will throw an error. Also, this is not the problem that I'm trying to address with my PR. My goal is to simplify the way we compute the Python version that is being used.

```
>>> major, minor = (2, 7)
>>> (major != 3 or major ==3) and minor < 6
False
>>> major != 3 or major ==3 and minor < 6
True
```

**Comment by TheYahya** [comment]
Thanks

**Pull request merged**